### PR TITLE
XSS tests

### PR DESCRIPTION
**Pull request by rjt-gupta**
Increased XSS emulator code coverage to 100%.

### CODE REVIEW

File: tanner/tests/test_xss_emulator.py [hunk 1]
```diff
@@ -11,6 +11,12 @@ def setUp(self):
         asyncio.set_event_loop(None)
         self.handler = xss.XssEmulator()
 
+    def test_scan(self):
```
Comment (afeena) [inline tanner/tests/test_xss_emulator.py]: please add also test with negative result

**Comment by rjt-gupta** [comment]
Added the test
